### ISSUE TITLE
DetailsList narrator announcement issues

### ISSUE THREAD

**Issue by satbai** [OfficeDev/office-ui-fabric-react#4240]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: VSTS

#### Describe the issue:

When selecting the first row in the list, the narrator doesn't announce that it was selected.

#### Actual behavior:

1. Go to https://developer.microsoft.com/en-us/fabric#/components/detailslist 
2. In the second example, navigate with the keyboard to the first row. The row is not selected.
![image](https://user-images.githubusercontent.com/24945574/37307893-8ce5782a-25f9-11e8-9bac-2f17b51c8323.png)
3. Click the space key to select the row
4. Narrator doesn't announce that the row was selected
![image](https://user-images.githubusercontent.com/24945574/37307940-b2881ce0-25f9-11e8-8cc8-804a1e78a0ad.png)

#### Expected behavior:

4. narrator should announce that the row was selected like it does for the next selections after the first one
![image](https://user-images.githubusercontent.com/24945574/37308028-f7d10690-25f9-11e8-9990-d29e207838e7.png)

**Comment by dzearing** [comment]
I am seeing `aria-selected` being toggled on the row. Is there something missing?

**Comment by satbai** [comment]
@cschlechty Is the first selection working as expected with screen readers? Not sure what's causing the issue since the DOM seems to get updated the same way in both cases; when the Narrator announces 'selected' and when it doesn't.

**Comment by satbai** [comment]
There's also a somewhat related narrator bug on the same component. IsSelectionItemPatternAvailable is true on all of the cells even when selection is not turned on. This can be reproduced in https://developer.microsoft.com/en-us/fabric#/components/detailslist in the first example. I'm using Inspect tool, Narrator and NarratorBuddy.

Repro steps:
1. Navigate to a row in a DetailList that doesn't have selection enabled. 
2. Observe individual cells with Inspect tool and see what the Narrator announces
3. Narrator announces 'selected' or 'non-selected' on the cell items 

![image](https://user-images.githubusercontent.com/24945574/37314601-42c9285e-2612-11e8-83f0-a8822c059e5e.png)

The element shouldn't announce 'selected' if selection is not enabled.

Please let me know if you think I should file a separate bug on this or update the bug description.

**Comment by ThomasMichon** [comment]
It looks like we should be able to use the `aria-multiselectable` attribute to indicate whether the list supports selection at all.

**Comment by raghavendra-kodimala** [comment]
@satbai can you please tell me where to download Narrator buddy. I want to view the text what Microsoft Narrator speaks. I have searched for it, but didn't found. It would be appreciable of you give us the download link or process to get the Narrator buddy.

**Comment by cschlechty** [comment]
Try aka.ms/narratorbuddy

**Comment by raghavendra-kodimala** [comment]
Thank you so much @cschlechty

**Comment by xugao** [comment]
closing this issue as it's being tracked in #6525
